foo.txt

```diff
@@ -0,0 +1 @@
+Test Xyz.
```

Comment:
- To consider:
  - [ ] Newer task on diff (uncompleted)
  - [X] Newer task on diff (completed)